src/components/form/select/index.js

```diff
@@ -0,0 +1,27 @@
+import { string, func } from 'prop-types';
+import cn from 'classnames';
+import './styles.css';
+
+const Select = ({ register, options, name, error, placeholder, className = '', ...rest }) => (
```

Comment:
Consider using a UI library for these common components. Later on you'll want to customize the option items, for example. Good options are PrimeReact, Material UI, Bootstrap.
<img width="288" alt="image" src="https://user-images.githubusercontent.com/38699660/162986994-b016955b-8f12-44a6-9ce2-bf69e211d9b6.png">

Reply:
Great, yes we talked about this, next tickets will be implementing with some library